### ISSUE TITLE
Add-MgTeamMember adds member but outputs error

### ISSUE THREAD

**Issue by JakeMoe** [microsoftgraph/msgraph-sdk-powershell#1277]
Using this code:

```
$bodyParameter = @{
  Values = @(
    @{
      '@odata.type' = 'microsoft.graph.aadUserConversationMember'
      Roles = @()
      'user@odata.bind' = "https://graph.microsoft.com/v1.0/users('$($user.Id)')"
    }
  )
}
Add-MgTeamMember -TeamId $teamId -BodyParameter $bodyParameter
```

adds the user $user.Id to the Team $teamId, but outputs this error:

> Add-MgTeamMember_Add1: Unable to cast object of type 'Microsoft.Graph.PowerShell.Runtime.Json.JsonObject' to type 'Microsoft.Graph.PowerShell.Runtime.Json.JsonArray'.

**Comment by JakeMoe** [comment]
I've noticed the error seems the same as #1298, which mentions #1159, which was closed as duplicate of #665.  Is this the same error, with a fix due ["soon"](https://github.com/microsoftgraph/msgraph-sdk-powershell/issues/1159#issuecomment-1086316422)?  If so, should this (and 1298) be closed as a duplicate as well?

Note that I'm also getting this error from Invoke-MgTimeUserOutlook now.

**Comment by peombwa** [comment]
Thanks for bringing this to our attention. The error is due to how the response schema is defined in the OpenAPI files as described in https://github.com/microsoftgraph/microsoft-graph-devx-api/issues/571. We are in the final stages of validating the fix, which will go out as part of this month's release.

Closing as duplicate of #665.
